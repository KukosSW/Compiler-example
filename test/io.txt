DECLARE
    a, b, t(0:10), tab(1000:2000)
BEGIN
    READ a;
    READ b;
    READ t(b);

    READ t(0);
    READ tab(1999);
    READ tab(a);

    WRITE a;
    WRITE b;
    WRITE t(b);
    WRITE 123;
    WRITE 1234567890;
    WRITE t(0);
    WRITE tab(1999);
    WRITE tab(a);
END
