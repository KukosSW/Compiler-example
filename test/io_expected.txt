5
1111
99
123
1234567890
88
77
66
